### PR TITLE
marco/feature-NFTRWP-189

### PR DESCRIPTION
**Pull request by marco-calderon-primelab**
- Added new property for ContactPopup component for showing a loading when gift is being sent.
- Added new state on sendNft component to enable showing of the loader. Only will show when request is being sent properly to the back end.

Evidence attached:

image

### CODE REVIEW

File: src/Components/Dashboard/SendNFT/sendNft.module.css [hunk 1]
```diff
@@ -143,7 +143,6 @@
   color: #2f80ed;
   font-size: 20.4px;
   font-weight: normal;
-  margin-left: 20px;
```
Comment (biswanathsahoo) [inline src/Components/Dashboard/SendNFT/sendNft.module.css]: Do this change in sendNft.module.scss file

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]
Please do a rebase with dev branch and resolve the conflict.

**Comment by marco-calderon-primelab** [comment]
Hi @biswanathsahoo please review the changes on this branch, make the required adjustments.

**Comment by biswanathsahoo** [comment]
@marco-calderon-primelab Your changes breaking the import contact flow. After importing contact, it is displaying all the contacts in a list but the Gift NFT button is not appearing. Please check the flow once.

**Pull request closed without merging**